### ISSUE TITLE
how to train

### ISSUE THREAD

**Issue by goodlookingbear** [MFaceTech/InstantID#10]
Hello, may I ask how to create a JSON file? Given a dataset, how should I handle it to start training? I have seen using Blip to create text descriptions, what should I do next? Thank you very much!

**Comment by chenxinhua** [comment]
The JSON file format is as shown in the figure below, where face detection and keypoint detection can be done using RetinaFace, face feature extraction is done using InsightFace, and text labels are generated using Blip.

<img width="928" alt="image" src="https://github.com/user-attachments/assets/29efd037-32a8-4379-8a0e-90777b92fb81">

**Comment by jaebbb** [comment]
I trained 0.7M portrait images on 2 epochs using the same training code, but the identity is not preserved. What's the problem?
